### PR TITLE
Prepare docs for Aug 2023 1st wave of Providers

### PR DESCRIPTION
**Pull request by eladkal**
---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: airflow/providers/apache/sqoop/CHANGELOG.rst [hunk 1]
```diff
@@ -40,6 +40,19 @@ are no longer accepted as arguments for those methods. These should instead be p
 as ``extra_options`` while initializing the Hook or via ``extra_options`` parameter to the
 operator which instantiates the hook with those given ``extra_options`` dictionary.
 
+* ``Validate SqoopHook connection string and disable extra options from public hook methods (#33039)``
+
+Bug Fixes
+~~~~~~~~~
+
+* ``Validate connection host field for Sqoop connection (#32968)``
```
Comment (pankajkoti) [inline airflow/providers/apache/sqoop/CHANGELOG.rst]: I think we can remove this part as this was changed later in above PR #33039

File: airflow/providers/apache/hive/CHANGELOG.rst [hunk 2]
```diff
@@ -27,6 +27,14 @@
 Changelog
 ---------
 
+6.1.4
+.....
+
+Misc
+~~~~
+
+* ``Bring back mysql-connector-python as required depednency (#32989)``
```
Comment (pankajkoti) [inline airflow/providers/apache/hive/CHANGELOG.rst:36]:
```suggestion
* ``Bring back mysql-connector-python as required dependency (#32989)``
```

File: airflow/providers/celery/CHANGELOG.rst [hunk 3]
```diff
@@ -27,6 +27,15 @@
 Changelog
 ---------
 
+3.3.1
+.....
+
+Misc
+~~~~
+
+* ``aDd documentation generation for CLI commands from executors (#33081)``
```
Comment (hussein-awala) [inline airflow/providers/celery/CHANGELOG.rst:36]:
```suggestion
* ``Add documentation generation for CLI commands from executors (#33081)``
```

**Review by potiuk — APPROVED** [review]

File: airflow/providers/cncf/kubernetes/CHANGELOG.rst [hunk 4]
```diff
@@ -27,6 +27,29 @@
...
+
+* ``Fix waiting the base container when reading the logs of other containers (#33127)``
+* ``Fix: Configurable Docker image of 'xcom_sidecar' (#32858)``
+* ``Fix 'KubernetesPodOperator' sub classes default container_logs (#33090)``
+* ``Consider custom pod labels on pod finding process on 'KubernetesPodOperator' (#33057)``
+
+Misc
+~~~~
+
+* ``aDd documentation generation for CLI commands from executors (#33081)``
```
Comment (hussein-awala) [inline airflow/providers/cncf/kubernetes/CHANGELOG.rst:45]:
```suggestion
* ``Add documentation generation for CLI commands from executors (#33081)``
```

**Review by hussein-awala — APPROVED** [review]

File: airflow/providers/apache/hive/CHANGELOG.rst (hunk 2 above)
Comment (eladkal) [reply airflow/providers/apache/hive/CHANGELOG.rst:36]: We can not change commit messages in provider releases :(

File: airflow/providers/celery/CHANGELOG.rst (hunk 3 above)
Comment (eladkal) [reply airflow/providers/celery/CHANGELOG.rst:36]: same as https://github.com/apache/airflow/pull/33128#discussion_r1284828409

File: airflow/providers/cncf/kubernetes/CHANGELOG.rst (hunk 4 above)
Comment (eladkal) [reply airflow/providers/cncf/kubernetes/CHANGELOG.rst:45]: same as https://github.com/apache/airflow/pull/33128#discussion_r1284828409

File: airflow/providers/apache/hive/CHANGELOG.rst (hunk 2 above)
Comment (potiuk) [reply airflow/providers/apache/hive/CHANGELOG.rst:36]: We can change them in CHANGELOG in fact. We can't change them in commits.rst but in changelog they are matched by PR # not the message.

**Pull request merged**